src/evergreen_names.rs

```diff
@@ -99,3 +107,7 @@ pub const MULTIVERSION_EXCLUDE_TAGS_FILE: &str = "multiversion_exclude_tags.yml"
 pub const MULTIVERSION_LAST_LTS: &str = "last_lts";
 /// Name of last continuous configuration.
 pub const MULTIVERSION_LAST_CONTINUOUS: &str = "last_continuous";
+
+// Distros
+/// Name of compile task distro for burn_in_tags.
+pub const COMPILE_TASK_DISTRO: &str = "rhel80-xlarge";
```

Comment:
I don't like this, but it was hardcoded this way in burn_in_tags.py and I didn't find a way to get this value from somewhere else.

We can add a mandatory buildvariant expansion for burn_in_tags, like we already have `burn_in_tag_buildvariants` expansion: https://github.com/mongodb/mongo/blob/master/etc/evergreen.yml#L1186
We can add `burn_in_tag_compile_distro`, but this will work only if we want to have them all to be the same distro. 

This is actually should be a map `burn_in_tag_build_variant_name: burn_in_tag_compile_distro`.

Reply:
Can you cut a ticket to describe that issue and track the future work?

Reply:
Sure, will do after merging this.